sigs.yaml

```diff
@@ -1906,6 +1937,7 @@ workinggroups:
       mailing_list: https://groups.google.com/forum/#!forum/kubernetes-wg-apply
   - name: Cloud Provider
     dir: wg-cloud-provider
+    charter_link:
```

Comment:
are working groups supposed to have a charter? https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md#sig-charter-guide says that only SIGs need to have one.

Reply:
if WGs are also supposed to have a charter, can we add the `charter_link` field for all WGs for consistency? this only adds for wg-cloud-provider.

- wg-apply has a comment which says "we might want to add a charter here".
- wg-app-def has a gdoc as their charter in the mission statement.